RadixWallet/Features/DappInteractionFeature/Children/Login/Row/PersonaRow+View.swift

```diff
@@ -22,53 +22,89 @@ extension PersonaRow {
 	@MainActor
 	struct View: SwiftUI.View {
 		let viewState: ViewState
-		let isSelected: Bool
-		let action: () -> Void
+		let mode: Mode
```

Comment:
Changes in this view are basically allowing to have `selection` mode (as it was currently used), and `display` mode (which doesn't show a `RadioButton` on the trailing side nor embeds the `View` inside a `Button`.